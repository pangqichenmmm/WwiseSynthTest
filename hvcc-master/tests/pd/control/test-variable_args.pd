#N canvas 632 324 312 269 10;
#X obj 66 102 variable_args 100 dog 5;
