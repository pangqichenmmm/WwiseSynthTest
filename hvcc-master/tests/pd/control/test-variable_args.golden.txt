[@ 0.000] print: 15
